src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx

```diff
@@ -25,19 +27,15 @@ Because of this, ensure your firewall allows their network requests through it.
 
 ## IP addresses are not personal data [#not-personal-data]
 
-Minions are deployed on servers, and the agents are expected to be activated using non-personal credentials. IP addresses associated with minions running on servers are not personal data under data protection and privacy laws. For more information, see the [Synthetic's security documentation](/docs/synthetics/new-relic-synthetics/getting-started/security-new-relic-synthetics).
+Minions are deployed on servers, and the agents are activated using non-personal credentials. IP addresses associated with minions running on servers are not personal data under data protection and privacy laws. For more information, see the [Synthetic's security documentation](/docs/synthetics/new-relic-synthetics/getting-started/security-new-relic-synthetics).
```

Comment:
This wording can change the meaning pretty significantly. Do the users provide the credentials? If so, I would probably keep "expected to be" as it implies the best practice is to use non-personal credentials. If we control the credentials, this edit is fine.

Reply:
!! Good catch! I'll change it back to 'expected to be.' I removed it to streamline the sentence and didn't realize meaning would be affected ;)